src/content/en/2020/markup.md

```diff
@@ -338,7 +321,7 @@ Nothing changed in the Top 7, but the `option` element went a little out of favo
 
 #### `details` and `summary`
 
-Another thing we were curious about was the use of the [`details` and `summary` elements](https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements), especially since 2020 brought [broad support](https://caniuse.com/details). Are they being used? Are they attractive for—even popular—among authors? As it turns out, only 0.39% of all tested pages are using them, although it's hard to gauge whether they were all used the correct way in exactly the situations when you need them, "popular" is the wrong word.
+Another thing we were curious about was the use of the [`details` and `summary` elements](https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements), especially since 2020 brought [broad support](https://caniuse.com/details). Are they being used? Are they attractive for—even popular—among authors? As it turns out, only 0.39% of all tested pages are using them—although it's hard to gauge whether they were all used the correct way in exactly the situations when you need them.
```

Comment:
No, that doesn’t work yet—with this edit, “although” doesn’t fit anymore. One could remove that last part (after the em dash), but then it takes away our interpretation. I suggest to keep this as it was.

A side note, “using these elements” could be a useful edit as “using them” starts to get unclear. If you could edit that then I wouldn’t need to track it (I had reviewed the chapter independently and am preparing another—small—PR).

Reply:
I have to read that first sentence several times to understand it - particularly with the last `"popular is the wrong word." bit.

What about dropping that but reverting the em-dash to a comma?

```suggestion
Another thing we were curious about was the use of the [`details` and `summary` elements](https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements), especially since 2020 brought [broad support](https://caniuse.com/details). Are they being used? Are they attractive for—even popular—among authors? As it turns out, only 0.39% of all tested pages are using them, although it's hard to gauge whether they were all used the correct way in exactly the situations when you need them.
```

Reply:
Actually, I think the em dash works—in the old setting. What do you think of the old version but with an em dash instead of a comma? (In my view it helps.)

Reply:
```suggestion
Another thing we were curious about was the use of the [`details` and `summary` elements](https://html.spec.whatwg.org/multipage/rendering.html#the-details-and-summary-elements), especially since 2020 brought [broad support](https://caniuse.com/details). Are they being used? Are they attractive for—even popular—among authors? As it turns out, only 0.39% of all tested pages are using them—although it's hard to gauge whether they were all used the correct way in exactly the situations when you need them—"popular" is the wrong word.
```